bundle/manifests/metallboperator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Do we need this?
I don't believe the operator is exposing metrics

Reply:
was created automatically in the initial setup, decided to keep it for now.
https://github.com/metallb/metallb-operator/blob/main/config/rbac/auth_proxy_service.yaml

Reply:
but if you don''t expose metrics there is no need for that

Reply:
you can remove it from the kustomize and leave the file for future use